lib/Client.php

```diff
@@ -203,4 +305,42 @@ private function message($msg)
 
         return $msg;
     }
+
+    private function loadFeatureFlags(): void
+    {
+        $response = $this->httpClient->sendRequest(
+            '/api/feature_flag',
```

Comment:
We've made a change since you last edited this, so this should now be `/api/feature_flag?token=project_api_key_here`.

No way you could have known though!